packages/@dcl/inspector/src/lib/data-layer/host/utils/migrations/fix-network-entity-values.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, IEngine } from '@dcl/ecs'
+import { CoreComponents, SdkComponents } from '../../../../sdk/components'
+import { INSPECTOR_ENUM_ENTITY_ID_START } from '../../../../sdk/enum-entity'
+
+/**
+ * Fixes duplicate entity IDs in NetworkEntity components within the engine.
+ *
+ * This function scans through all entities that have the NetworkEntity component
+ * and identifies any duplicate entity IDs. If duplicates are found, it reassigns
+ * unique entity IDs to each NetworkEntity component to ensure all entity IDs are unique.
+ */
+export function fixNetworkEntityValues(engine: IEngine) {
+  const NetworkEntity = engine.getComponentOrNull(CoreComponents.NETWORK_ENTITY) as SdkComponents['NetworkEntity']
+
+  function needsFixing(): boolean {
+    const usedEntities: Set<Entity> = new Set()
+    for (const [_, value] of engine.getEntitiesWith(NetworkEntity)) {
+      if (usedEntities.has(value.entityId)) {
+        return true
```

Comment:
based on the previous question, could we use the method `getNextEnumEntityId` here?

Reply:
Yes, that was my first approach but the algorithm for that is to iterate all the entities to look for entities not being used.
It was simpler to just re-generate them IMO